ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -1260,7 +1271,7 @@ let getHeadmarkerId = (data, matches) => {
       alertText: {
         en: 'Close Tethers',
         de: 'Nahe Verbindungen',
-        fr: 'Liens proches',
+        fr: 'Rapprochez les liens',
```

Comment:
If the mechanics are to bring closer the links, my translation is better. If I didn't understand correctly because I watched the video, tell me. I may be wrong or have misunderstood.